website/docs/cli/cloud/settings.mdx

```diff
@@ -42,87 +63,91 @@ terraform {
 
-### Environment Variables
+As a result, Terraform prompts you to rename workspaces according to a pattern relative to their existing names. The pattern is intended to indicate that the workspaces share configuration. A common strategy is `<COMPONENT>-<ENVIRONMENT>-<REGION>`, for example `networking-prod-us-east` and `networking-staging-us-east`. Refer to [Workspace Naming](/terraform/cloud-docs/workspaces/naming) in the HCP Terraform documentation for additional information.
 
--> **Note:** CLI integration environment variables are supported in Terraform v1.2.0 and later.
+### Migrate remote backend
 
-You can use environment variables to configure one or more `cloud` block attributes. This is helpful when you want to configure Terraform as part of a Continuous Integration (CI) pipeline. Terraform only reads these variables if the corresponding attribute is omitted from your configuration file. If you choose to configure the `cloud` block entirely through environment variables, you must still add an empty `cloud` block in your configuration file.
+In the `terraform` block or `terraform.tf` file, replace `backend "remote"` with `cloud`. Terraform will continue to use the same ste of HCP Terraform workspaces.
 
-~> **Warning:** Remote execution with non-interactive workflows requires auto-approved deployments. Minimize risk of unpredictable infrastructure changes and configuration drift by making sure that no one can change your infrastructure outside of your automated build pipeline.  Refer to [Non-Interactive Workflows](/terraform/cloud-docs/run/cli#non-interactive-workflows) for details.
+The following example migrates the state data for a single workspace:
 
-Use the following environment variables to configure the `cloud` block:
+```hcl
+terraform {
+-  backend "remote" {
++    cloud {
+       organization = "my-org"
+
+       workspaces {
+          name = "my-app-prod"
+       }
+     }
+   }
+}
+```
 
-- `TF_CLOUD_ORGANIZATION` - The name of the organization. Terraform reads this variable when `organization` omitted from the `cloud` block`. If both are specified, the configuration takes precedence.
+If the `terraform` block or `terraform.tf` file uses the `prefix` argument to connect to multiple workspaces, you can specify a list of single-value string tags in the `tags` argument instead. During `terraform plan` or `terraform apply` operations, Terraform associates the configuration with workspaces that match the specified tags.
 
-- `TF_CLOUD_HOSTNAME` - The hostname of a Terraform Enterprise installation. Terraform reads this when `hostname` is omitted from the `cloud` block. If both are specified, the configuration takes precedence.
+The following example replaces the `my-app-` prefix with the `app:mine` tag:
```

Comment:
The code blocks have `-` and `+` to indicate how to change the file. I take your point, but I think because 1) we're making a concerted effort to stop referring to previous examples moving forward and 2) that when you see the example block that it will be clear. We can revisit if people struggle to understand.